examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL2/SDL.h>
+
+int main(int argc, char *argv){
```

Comment:
It needs to inject on anything that the native entry point is not main(), which is pretty much everything except unix based Oses

> I've just tried simple example with char** argv and it works fine with Visual Studio 2013.

Did you link with SDLmain?